### PR TITLE
Update Module 6

### PR DESCRIPTION
**Pull request by simonkurtz-MSFT**
- Format, clarify, standardize Module 6
- Minor modification of Azure Subscription ID in Module 1 as part of the Module 6 work

### CODE REVIEW

File: docs/aca/06-aca-dapr-bindingsapi/index.md [hunk 1]
```diff
@@ -449,39 +459,44 @@ az containerapp env dapr-component set `
 
 Next, we will add create the three Dapr bindings components using the component files created.
 
-!!! warning "Important"
+<!-- 10/23/2023 - Simon Kurtz - Commenting out as the current AZ CLI version does not error, but I am not yet ready to remove it (even though it is in source control history) -->
```
Comment (tjoudeh) [inline docs/aca/06-aca-dapr-bindingsapi/index.md:462]: If you didn't face this issue then the CLI is updated, the issue opened still not closed. We can remove this warning

**Review by tjoudeh — APPROVED** [review]
Looks good, just the comment on CLI known issue which seems it is resolved by now.

**Review by waelkdouh — APPROVED** [review]
Lgtm

File: docs/aca/06-aca-dapr-bindingsapi/index.md (hunk 1 above)
Comment (simonkurtz-MSFT) [reply docs/aca/06-aca-dapr-bindingsapi/index.md]: Affirmative that the CLI now addresses this correctly. I just removed the commented out warnings in the latest commit.

**Pull request merged**